CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for
 # review when someone opens a pull request.
-*       @schedulaar @RoAnBu @elKei24 @maxammann @aiquita @PaulaWi
+*       @schedulaar @RoAnBu @elKei24 @maxammann @aiquita @PaulaWi @klinzo
```

Comment:
I think you can remove all except for @schedulaar, @klinzo and @maxammann

At least @aiquita wants to be removed afaik